### PR TITLE
kvserver: add option to skip consistency queue

### PR DESCRIPTION
**Pull request by stevendanna**
Epic: none
Release note: None

### CODE REVIEW

File: pkg/kv/kvserver/consistency_queue.go [hunk 1]
```diff
@@ -46,6 +46,16 @@ var consistencyCheckRate = settings.RegisterByteSizeSetting(
...
 	settings.WithPublic)
 
+// skipConsitencyQueueForExternalBytes is a setting that controls whether
+// replicas with external bytes should be processed by the consitency
+// queue.
+var skipConsitencyQueueForExternalBytes = settings.RegisterBoolSetting(
+	settings.SystemOnly,
+	"server.consistency_check.skip_external_bytes.enabled",
+	"skip the consistency queue for external bytes",
+	false,
```
Comment (dt) [inline pkg/kv/kvserver/consistency_queue.go:56]: can we make this true?

**Review by dt — APPROVED** [review]

File: pkg/kv/kvserver/consistency_queue.go (hunk 1 above)
Comment (dt) [reply pkg/kv/kvserver/consistency_queue.go:56]: actually we can do that in a follow-up that also gets the merge queue

**Comment by stevendanna** [comment]
TFTR!

bors r=dt

**Comment by dt** [comment]
blathers backport 24.1

**Pull request merged**